pkg/rpcservice/trigger.go

```diff
@@ -20,7 +21,19 @@ func (srv *RPCService) Trigger(ctx context.Context, req *rpc.TriggerRequest) (*r
 		return nil, status.Error(codes.Unauthenticated, err.Error())
 	}
 
-	requestNames, err := fixtures.Trigger(req.Event, req.StripeAccount, baseURL, apiKey)
+	fmt.Println("EVELYN - pkg/rpcservice/trigger.go:Trigger")
+	fmt.Println(req)
+
```

Comment:
Please remove these print statements. This info is logged at https://github.com/stripe/stripe-cli/blob/master/pkg/rpcservice/middleware.go#L55-L57.

Reply:
oops! thanks for catching it